tests/page_test.go

```diff
@@ -1699,3 +1699,59 @@ func TestShadowDOMAndDocumentFragment(t *testing.T) {
 		})
 	}
 }
+
+func TestPageTargetBlank(t *testing.T) {
+	t.Parallel()
+
+	tb := newTestBrowser(t, withHTTPServer())
+	tb.withHandler("/home", func(w http.ResponseWriter, _ *http.Request) {
+		_, err := w.Write([]byte(
+			`<!DOCTYPE html><html><head></head><body>
+				<h1>click please</h1><a href="/link" target="_blank">click me</a>
+			</body></html>`,
+		))
+		require.NoError(t, err)
+	})
+	tb.withHandler("/link", func(w http.ResponseWriter, _ *http.Request) {
+		_, err := w.Write(
+			[]byte(`<!DOCTYPE html><html><head></head><body><h1>you clicked!</h1></body></html>`),
+		)
+		require.NoError(t, err)
+	})
+
+	p := tb.NewPage(nil)
+
+	// Navigate to the page with a link that opens a new page.
+	opts := &common.FrameGotoOptions{
+		Timeout: common.DefaultTimeout,
+	}
+	resp, err := p.Goto(tb.url("/home"), opts)
+	require.NoError(t, err)
+	require.NotNil(t, resp)
```

Comment:
Why do we need `resp`?

Reply:
We don't. It's just to ensure that it is there. I would remove it, but it don't think it hurts to leave it in. Unless there's a reason i'm missing?